reason-parser/src/reason_pprint_ast.ml

```diff
@@ -3364,39 +3437,60 @@ class printer  ()= object(self:'self)
      token "+", in `x + a * b` should not reduce until after the a * b gets
      a chance to reduce. This function would determine the minimum parens to
      ensure that. *)
-  method ensureContainingRule ~withPrecedence ~reducesAfterRight =
+  method ensureContainingRule ~withPrecedence ~reducesAfterRight () =
```

Comment:
Yes, it has become something I always write. It makes it so much easier when revisiting/extending the code at a later point. No point in making life harder.